### DISCUSSION TITLE
We made a website for all F1 related analysis including every season Telemetry data in 3D.

### DISCUSSION THREAD

**Discussion by adityakotha03** [br-g/openf1#68] [Show and tell]
Are you ready to take your F1 analysis to the next level? Introducing F1nsight, the ultimate website for all your F1-related needs. Whether you're a die-hard fan or a casual viewer, F1nsight has something for everyone.

At F1nsight, we've created a one-stop destination for in-depth analysis of every F1 race session from the 2023 season and beyond. But that's just the tip of the iceberg. Our platform goes beyond mere race recaps and standings. We've built an immersive 3D experience that allows you to explore telemetry data like never before.

![cons](https://github.com/br-g/openf1/assets/86183042/9a566a63-dded-42b0-b90c-ecf03a890157)

Excited yet? Here's what you can expect from F1nsight:

1. Comprehensive Analysis: Dive deep into every aspect of the race, from lap times to tire strategies and more. Our detailed leaderboards provide comprehensive rankings and statistics from previous races, giving you a complete picture of each event.

2. Interactive Telemetry Viewer: Get up close and personal with your favorite drivers' telemetry data as they navigate the track. Our interactive viewer lets you follow their race unfold lap by lap, with multiple camera views and detailed telemetry data at your fingertips.

3. Mobile-Friendly Design: While our site is optimized for larger screens, all components are fully mobile-friendly. So whether you're at home or on the go, you can access F1nsight anytime, anywhere.

But we're not done yet. F1nsight is a community-driven project, and we welcome contributions from fellow F1 enthusiasts like you. Our repository is fully open-sourced on GitHub (https://github.com/adityakotha03/F1nsight), so you can dive into the code, report issues, and even contribute your own ideas to make F1nsight even better.

So what are you waiting for? Head over to https://www.f1nsight.com/ and start exploring the world of F1 like never before. With F1nsight, the thrill of the race is just a click away.
![image](https://github.com/br-g/openf1/assets/86183042/04dab802-e3d2-42a2-bfa0-2a11cef1ecab)
![image (1)](https://github.com/br-g/openf1/assets/86183042/e1da110c-8fba-4495-b391-4173d080b369)

![Uploading clideo_editor_72f4c3320cb84703bdad0cb60887bd68.gif…]()

**Comment by adityakotha03** [comment]
Thank you so much, @br-g, for making this website possible with the OpenF1 API; this project would not have been possible without this. He has also fixed many issues we ran into, so we are very thankful for his community support. Also, without @ant-dot-comm, this would not be possible at all. I am grateful we worked together on this project.

**Comment by br-g** [comment]
The app looks great, I will definitely give it a try :smiley:
Thank you for sharing the project!

**Comment by mohamedhadrami** [comment]
This looks incredible. Great job guys!
